Documentation/Design/lambda_body_design.md

```diff
@@ -23,30 +24,29 @@ lambda x, y: x == y
 
 ### Ruby:
 ```Ruby
-lambda { |x, y|  x == y}
+lambda { |x, y|  x == y }
 ->(x, y) { x == y }
 ```
 Both of the above constructs are valid for Ruby. GLS generates the top one for no reason at all.
 
 ### TypeScript
 ```TypeScript
-(x, y) => x == y
+(x, y) => x === y
 ```
 
 
 ## Design
+
 ### GLS Syntax:
 `lambda : `*`[parameterName, parameterType, ...]`* `command`
```

Comment:
@JoshuaKGoldberg Should we also require return type in case that is needed as well, because we are requiring parameter types?